spec/components/buttons_component_spec.rb

```diff
@@ -6,12 +6,22 @@
 RSpec.describe ButtonsComponent do
   let(:component) { described_class.new(form: form) }
   let(:form) { ActionView::Helpers::FormBuilder.new(nil, work_form, controller.view_context, {}) }
-  let(:work) { build(:work, collection: build(:collection, id: 7)) }
+  let(:work) { build(:work, collection: create(:collection, id: 7)) }
```

Comment:
given the changes in `buttons_component.rb`, the collection needs to be in the db for the `.find` call

Reply:
with the suggestion I gave above, this approach would not be required.